ibm/mas_devops/roles/suite_app_config/vars/predict.yml

```diff
@@ -5,3 +5,15 @@ mas_app_ws_kind: PredictWorkspace
 
 mas_app_cfg_delay: 120
 mas_app_cfg_retries: 30
+
+mas_app_settings_wml_version: "{{ lookup('env', 'WML_VERSION') | default('4.5', true) }}"
+mas_app_settings_watsonstudio_project_id: "{{ lookup('env', 'CPD_PREDICT_WS_PROJECT') | default('', true) }}"
+mas_app_settings_imagePullPolicy: "{{ lookup('env', 'MAS_IMAGE_PULL_POLICY') | default('IfNotPresent', true) }}"
+
+# CPD version is required by predict to identify what version of WML must be used to activate predict
+# if this information is used by more than predict, it should be moved to defaults/main.yml of the role
+cpd_product_version: "{{ lookup('env', 'CPD_PRODUCT_VERSION') }}"
```

Comment:
this is already in the defaults/main.yaml (line 24) is there a reason for it to be here?